servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -380,8 +382,17 @@ private static CommitMeta addHashToCommitMeta(Hash hash, CommitMeta commitMeta)
     return logEntries.filter(
         logEntry -> {
           try {
+            List<Operation> operations = logEntry.getOperations();
+            if (operations == null) {
+              operations = Collections.emptyList();
+            }
+            // ContentKey has some @JsonIgnore attributes, which would otherwise not be accessible.
```

Comment:
Do we not control which attributes are visible to CEL expressions? Could we make `@JsonIgnore` attributes visible?